go.mod

```
@@ -23,6 +23,7 @@ require (
 	github.com/gin-contrib/sessions v0.0.5
 	github.com/gin-contrib/size v0.0.0-20230212012657-e14a14094dc4
 	github.com/gin-gonic/gin v1.9.1
+	github.com/go-ldap/ldap v3.0.3+incompatible
```

It looks like this library is up to v3.4.5, and this run of version is older (2019) from before they used the proper version suffix for the v3 module. Should we be using a more recent version?
```suggestion
	github.com/go-ldap/ldap/v3 v3.4.5
```

---

(all of the imports paths will have to be updated to v3 also)

---

Oh good call, I was able to update and test backwards compatibility 🙂